app.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+const app = express()
+const { getAllTopics } = require('./controller/controller.js')
+
+const topicsRegex = new RegExp(/\/api\/topics.*/g)
```

Comment:
what's this about? :)

Reply:
Was trying to make a regex so that if a user went to an endpoint such as '/api/topics2729279' or '/api/topics/sjsjs' it would default to '/api/topics' and ignore anything extra, wasn't able to figure it out for now so will go back to it later :)